MaioPluginSample/Assets/Plugins/Maio.cs

```diff
@@ -71,10 +77,9 @@ private class MaioAdsListener : AndroidJavaProxy {
             Maio.OnChangedCanShowEventHandler(zoneId, newValue);
         }
 
-        ////再生直前に呼ばれる処理
-        //public void onOpenAd(string zoneId) {
-        //    Maio.OnOpenAdEventHandler(zoneId);
-        //}
+        //再生直前に呼ばれる処理
+        public void onOpenAd(string zoneId) {
+        }
```

Comment:
This change is different from the purpose of the pull request and cannot be evaluated by me.
Please remove this once and handle it with another pull request.